File: backend/hiFes/src/main/java/hiFes/hiFes/dto/HostUserSignUpDto.java
package hiFes.hiFes.dto;


import lombok.Getter;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@Getter
public class HostUserSignUpDto {
    private String email;
    private String name;
    private String phoneNo;
    private String organization;
    private String orgNo;
    private String orgCode;
}

File: frontend/web/lib/dto/HostUserSignUpDto.dart
class HostUserSignUpDto {
  final String organization;
  final String orgCode;
  final String orgNo;
  final String phoneNo;

  HostUserSignUpDto({
    required this.organization,
    required this.orgCode,
    required this.orgNo,
    required this.phoneNo
  });

  // factory UserDTO.fromJson(Map<String, dynamic> json) {
  //   return UserDTO(
  //     id: json['id'],
  //     username: json['username'],
  //     email: json['email'],
  //   );
  // }

  Map<String, dynamic> toJson() {
    return {
      'organization': organization,
      'orgCode': orgCode,
      'orgNo': orgNo,
      'phoneNo': phoneNo,
    };
  }
}
